src/arch/arm/pagetable.cc

```diff
@@ -489,5 +489,14 @@ getPageTableOps(GrainSize trans_granule)
     }
 }
 
+TLBTypes::KeyType::KeyType(const TlbEntry &entry)
```

Comment:
For a future PR I would recommend fixing this. The TLB entry contains a key type which we want to copy. The proper solution would be to have a KeyType var in TlbEntry and to KeyType(entry.keyType).